src/cloud-resources/src/crd/materialize.rs

```diff
@@ -495,7 +495,7 @@ pub mod v1alpha1 {
                 if next_version.major != active_version.major {
                     if next_version.major == 26
                         && active_version.major == 0
-                        && active_version.minor == 164
+                        && active_version.minor >= 147
```

Comment:
On further thought it seems like the problem is that not all versions in v0.147 and >= v0.148 have the necessary patch for the forwards compatibility. Which is why we tell customers:

> 👉 Note: Before upgrading from v25.2 to v26.0, [upgrade first to version 25.2.16](https://materialize.com/docs/self-managed/v25.2/release-notes/#v25216). Afterwards, you can [upgrade to v26.0](https://materialize.com/docs/installation/#upgrade).

I think only v0.147.20 (or >= 20) and v0.164.X are fine.